daml-script/export/src/main/scala/com/daml/script/export/Encode.scala

```diff
@@ -398,14 +398,17 @@ private[export] object Encode {
       Doc.intercalate(Doc.text(", "), ps.map(encodeParty(partyMap, _))) +
       Doc.text("]")
 
-  private def encodeCid(cidMap: Map[ContractId, String], cid: ContractId): Doc = {
+  private def encodeCidExpr(cidMap: Map[ContractId, String], cid: ContractId): Doc = {
```

Comment:
we were using `encodeCid` (now `encodeCidExpr`) in pattern position, which would produce nonsensical code like

```haskell
  do
    (lookupContract "foo" contracts) <- submit ...
```

Reply:
:+1: